src/console_reporter.cc

```diff
@@ -92,7 +95,9 @@ void ConsoleReporter::PrintRunData(const Run& result) {
                    " items/s");
   }
 
-  double const multiplier = 1e9; // nano second multiplier
+  double const multiplier = time_unit_ == "ns" ? 1e9 : 1e3; // nano second or
```

Comment:
right. that's essentially what we do with the values that go into the benchmark name.

it might make at-a-glance comparison a little more awkward if a single benchmark jumps between units. i'm not sure what's best.